internal/utils.go

```diff
@@ -0,0 +1,48 @@
+package internal
```

Comment:
@lucasmenendez why this packages is called internal and not utils?

Reply:
It is called `internal` because this way go prevents any third party package from importing it, only the packages of the same project ([read more here](https://dev.to/stefanalfbo/internal-packages-in-go-2h9)). 

We can create another package inside the internal one called `utils` if you want, but for me it does not make sense at the moment because it is the only file inside the `internal` package.